host/sgx/internal_u_wrapper.c

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <openenclave/host.h>
+
+#include <openenclave/edger8r/host.h>
+#include <openenclave/internal/calls.h>
+#include <openenclave/internal/raise.h>
+
+/* Override oe_call_enclave_function() calls with _call_enclave_function(). */
+#define oe_call_enclave_function _call_enclave_function
+
+/* The ocall edge routines will use this function to route ecalls. */
+static oe_result_t _call_enclave_function(
```

Comment:
Rename to `_call_internal_enclave_function`?

Reply:
Good idea. Resolved.